src/main/kotlin/org/radarbase/redcap/config/RedCapInfo.kt

```diff
@@ -27,6 +27,7 @@ import java.net.URL
 data class RedCapInfo(
     @JsonProperty("url") var url: URL,
     @JsonProperty("project_id") val projectId: Int,
+    @JsonProperty("api_path") val apiPath: String = "/api/",
```

Comment:
Does this mean there are instances when it's not `/redcap/api`? Or will that go to the url now?

Reply:
I think it was that in our case since our redcap was available on path `/redcap` (possibly via reverse proxy) but that is not a part of original redcap url structure. So while in our case it was `/redcap/api`, actually if the server is deployed on root then the path would be just `/api`.

Reply:
But yes we would need to configure `/redcap/api` for the current redcap server but only a few projects using it and it is scheduled for sunsetting soon. `/api` as a default makes more sense to me.

Reply:
Ah okay I see. Thanks for explaining!